src/main/kotlin/no/skatteetaten/aurora/boober/service/vault/VaultService.kt

```diff
@@ -32,7 +33,12 @@ class VaultService(
     val userDetailsProvider: UserDetailsProvider
 ) {
 
-    private val logger = LoggerFactory.getLogger(VaultService::class.java)
+    fun findVaultKeys(vaultCollectionName: String, vaultName: String): List<String> {
```

Comment:
Regelen for hvilken fil som blir brukt er at latest.properties blir brukt dersom ikke annet er spesifisert. Dersom det f.eks. finnes en fil som heter 2.properties og din versjon er 2.etellerannet så er det denne som brukes (mener jeg :| )

Jeg tror ikke det blir helt riktig å se i alle filene, men jeg skal ta en ekstra verifikasjon på dette til mandag. Litt dumt å måtte duplisere den logikken her fra startscriptet...

Reply:
Hmm, ok. Det vil være dumt å måtte duplisere dette ja, så da er jeg litt usikker på hvordan jeg skal løse denne.